app/src/main/res/values/colors.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="purple_200">#FFBB86FC</color>
-    <color name="purple_500">#FF6200EE</color>
+    <color name="purple_200">#8C9DDF</color>
+    <color name="purple_500">#8C9DDF</color>
     <color name="purple_700">#FF3700B3</color>
```

Comment:
what are 200, 500, and 700? 200 and 500 look like they're now the same color, and they're pretty far from FF3700B3 which looks more pink to me

Reply:
Included my app color preference to remove the default color. Will work on this.